.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,102 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      elk-stack-version:
+        required: true
+        description: "Elastic Cloud stack version"
+        default: "8.8.0-SNAPSHOT"
+      ess-region:
+        required: true
+        description: "Elastic Cloud deployment region"
+        default: "gcp-us-west2"
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_REGION: "eu-west-3"
+  TF_VAR_ec_api_key: ${{ secrets.CI_EC_API_KEY }}
+  EC_DEPLOYMENT: "module.ec_deployment"
+  WORKING_DIR: deploy/cloud
+
+jobs:
+  Setup:
```

Comment:
Seems like we don't do anything on `Setup` besides outputting variables, in that case we can have a single job.

Reply:
`Setup` contains only outputting variables because it is an initial stage of workflow development. Adding scheduler will add a logic for all variables and in this case a separate `Setup` job will be better than to have a single job. In addition this flow will manage also deployment of KSPM and CSPM components.  Defining logic of what should be deployed in a separate `Setup` job is preferable than a single job for all.